TrackerConditions/test/MisalignTracker.C

```diff
@@ -26,9 +26,10 @@ double PanelPhi(int plane, int panel){
 }
 
 
-void MisalignTracker(int seed, double weakfactor, double trackerangle, double trackerpos,
+void MisalignTracker(int seed, bool misaligntracker, bool misalignplanes, bool misalignpanels, bool misalignstraws,
+    double weakfactor, double trackerangle, double trackerpos,
     double planeangle, double planepos,
-    double panelangle, double panelpos,
+    double panelangle, double panelposv, double panelposw, double rsqueeze,
     double wiredV, double wiredW,
```

Comment:
Is rsqueeze unrelated to panelposv?

Reply:
rsqueeze is the weak mode, it needs to be separate from the other weak modes since it has a different mechanism and scale